keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -1333,13 +1732,41 @@ will rollout across nodes.
 
 ###### What specific metrics should inform a rollback?
 
+A substantial increase in the `job_sync_duration_seconds` metric may suggest the
+processing of the configured job pod failure policy rules consumes too much time.
+
+An operator can also observe `job_pod_failure_total` to check if the reason count
```

Comment:
Looks like there is already job pod total metric that can be used to get a failure ratio? Can we make note of it here so anyone reading this can trivially know how to get the failure ratio?

Reply:
The currently existing metric called `job_pods_finished_total` does not provide the statistics on the job controller's applied action according to the pod failure policy. However, it seems reasonable to extend the existing metric for this purpose rather than to introduce a new one. I updated the KEP to suggest extension of the currently existing metric by the `failure_policy_action` label.